src/main/java/frc/robot/hardware/AbsoluteEncoder.java

```diff
@@ -18,8 +18,10 @@ public enum EncoderConfig {
         BackRightModule(4, false, -3.995, -4.686),
         //Arm Encoders (REV)
         ArmBase(1, true, Units.degreesToRadians(-74.6)),
-        ArmElbow(0, true, Units.degreesToRadians(233.6));
-        
+        ArmElbow(0, true, Units.degreesToRadians(233.6)),
+        // Buddy Balance Encoders (REV)
+        BuddyBalance(3, true, 0, 2.709); // placeholder offset
```

Comment:
Will we only have one?

Reply:
I believe so.  @riversry mark the placeholder with TODO so we know that we remember to get the offset in the competition bot.